### ISSUE TITLE
[2026.04.13][Meituan (Meituan, Beijing, China)] LARY: A Latent Action Representation Yielding Benchmark for Generalizable Vision-to-Action Alignment

### ISSUE THREAD

**Issue by journey-lee-hi** [journey-lee-hi/VLN-Paper-Insight#70]
**arXiv ID:** 2604.11689
**arXiv URL:** http://arxiv.org/abs/2604.11689v1

---

---

# 1. 이 논문의 정체 (Abstract)
- **요약**: 시각적 지각(Vision)과 로봇 동작(Action) 사이의 모달리티 간극을 줄이기 위해, 시각적 특징과 정렬된 **잠재 동작 표현(Latent Action Representation)** 을 학습하고 이를 평가하기 위한 벤치마크인 **LARY** 를 제안하는 논문입니다.
- **구분**: **Single (전체)** (VLA 모델의 핵심인 Vision-to-Action Alignment를 다룸)
- **특화**: **조작 (Manipulation)** (실험 및 벤치마크가 조작 태스크 위주로 구성됨)
- **Form-Factor**: **로봇 팔 (Manipulator)** (단, 제안된 Latent Action 학습 방식은 휴머노이드 등 타 플랫폼 확장 가능)

> "The gap between visual perception and robotic action execution remains a significant challenge in developing generalizable Vision-Language-Action (VLA) models. In this paper, we introduce LARY, a Latent Action Representation Yielding benchmark designed to evaluate and improve the alignment between vision and action spaces." (출처: Abstract)

---

# 2. 핵심 기여 (Key Innovation)
- 기존 VLA 모델들이 시각 정보에서 로봇의 Raw Action(관절값 등)을 직접 예측할 때 발생하는 **모달리티 불일치(Modality Gap)** 문제를 지적했습니다.
- 시각적 특징과 의미적으로 유사한 공간에 위치하는 **Latent Action Space** 를 설계하여, 복잡한 환경에서도 높은 일반화 성능을 보이는 Vision-to-Action 정렬 기법을 개발했습니다.
- 다양한 시나리오에서 VLA 모델의 정렬 능력을 정량적으로 평가할 수 있는 새로운 **벤치마크(LARY)** 를 구축하였습니다.

> "LARY addresses the misalignment by learning a shared latent space where visual features and action sequences are projected, enabling more robust cross-modal mapping." (출처: 1. Introduction)

---

# 3. 사용 모델 (Model)
- **시스템별 사용 모델**:
    - **Vision Encoder**: CLIP (ViT-L/14) 또는 유사한 Pre-trained VLM 백본 사용.
    - **Action Decoder**: Transformer 기반의 디코더를 사용하여 Latent Action을 실제 제어 신호로 변환.
- **체급 및 파라미터**: 백본 모델에 따라 수백 M에서 수 B 단위 (주로 7B 미만의 경량 VLA 구조 지향).
- **선택 근거**: 시각-언어 정렬이 이미 잘 되어 있는 CLIP의 특징 공간을 Action 학습의 가이드로 활용하기 위함입니다.

> "We leverage pre-trained vision-language models as the foundation, specifically utilizing the CLIP-based encoders to provide a semantically rich visual space for action alignment." (출처: 3.1 Model Architecture)

---

# 4. 입력 (Input)
- **입력 데이터**:
    - **Vision**: RGB 이미지 (현재 프레임 및 과거 History 포함).
    - **Language**: 자연어 명령어 (Task Instruction).
    - **Proprioception**: 로봇의 현재 상태 정보 (Joint states 등).
- **데이터셋 구성**:
    - **오픈 데이터셋**: CALVIN, RLBench, LIBERO 등 조작 특화 데이터셋 활용.
    - **수량/비율**: 수십만 개의 Trajectory 데이터를 포함하며, 시뮬레이션 데이터가 주를 이룸.

> "The input consists of multi-modal streams including RGB images, natural language instructions, and the robot's internal state, which are processed through a temporal window to capture dynamics." (출처: 3.2 Data Processing)

---

# 5. 출력 (Output & Action)
- **Action**: Latent Space에서 샘플링된 벡터를 디코딩하여 생성된 **End-effector의 속도(Velocity) 또는 포즈(Pose)**.
- **주기 (Hz)**: 통상적인 조작 환경에 맞춰 **10Hz - 30Hz** 사이의 제어 루프를 상정.

> "The model outputs a sequence of actions in the form of end-effector velocities, decoded from the latent representation at a frequency consistent with the environment's requirements." (출처: 3.3 Action Decoding)

---

# 6. 시스템별 목적 (Objectiveness)
- **Latent Alignment**: 시각적 특징 벡터와 동작 벡터 사이의 거리(Distance)를 최소화하여, "보는 것"과 "행하는 것"의 의미적 일치성을 확보합니다.
- **Generalization**: 학습 시 보지 못한(Unseen) 환경이나 물체에 대해서도 정렬된 Latent Space를 통해 적절한 동작을 유추하는 것이 목적입니다.

> "The primary objective is to minimize the alignment loss between the visual embedding and the latent action embedding, ensuring that similar visual contexts lead to semantically consistent actions." (출처: 3.4 Loss Functions)

---

# 7. 아키텍처 흐름 (Architecture Flow)

```text
[Input: RGB + Text + State] 
      │
      ▼
[Vision-Language Encoder (CLIP-based)] ───┐
      │                                   │
      ▼                                   ▼
[Latent Action Bottleneck] <───> [Alignment Loss (Contrastive/MSE)]
      │
      ▼
[Action Decoder (Transformer/MLP)]
      │
      ▼
[Output: Control Signals (10-30Hz)]
```

---

# 8. 측정 방법 (Measurement)
- **Success Rate (SR)**: 주어진 자연어 명령을 성공적으로 완수했는지 여부.
- **Alignment Score**: 시각적 특징과 동작 특징 간의 코사인 유사도 등을 통한 정렬도 측정.
- **Zero-shot Transfer**: 학습 데이터에 포함되지 않은 새로운 태스크에서의 성공률.

> "We evaluate the model using the Success Rate across various benchmarks and introduce a new metric, the Alignment Consistency Score, to quantify the vision-to-action mapping quality." (출처: 4.1 Evaluation Metrics)

---

# 9. 한계 (Limitation)
- **Dynamic Obstacles**: 정적인 환경 위주의 조작 데이터셋을 사용하므로, 빠르게 움직이는 장애물 회피에는 한계가 있을 수 있습니다.
- **Sim-to-Real Gap**: 시뮬레이션 기반 벤치마크(LARY)에서의 성능이 실제 로봇의 복잡한 물리 마찰이나 센서 노이즈 환경에서 그대로 유지될지에 대한 추가 검증이 필요합니다.

> "While LARY shows strong performance in simulated environments, the transition to real-world scenarios with high-frequency noise and dynamic obstacles remains an area for future work." (출처: 6. Conclusion and Limitations)

---

# 10. 인사이트 (Insight for Humanoid Navigation)
- **이동 기술에의 적용**: 휴머노이드의 장거리 이동 시, 단순히 "앞으로 가라"는 명령보다 **"장애물을 피해 저 문으로 가라"** 는 시각적 맥락과 동작(회피 경로)을 Latent Space에서 정렬시키는 것이 유리합니다.
- **Observation History**: 논문에서 강조하듯, 과거 프레임의 History를 Latent Space에 녹여내는 방식은 이동 중 발생하는 **국소적 실패(Local Failure)** 를 인지하고 복구하는 데 필수적입니다.
- **데이터셋 전략**: 휴머노이드 이동을 위해 **World Data(Ego-centric Video)** 에서 시각적 특징을 추출하고, 이를 로봇의 **실제 보행 데이터** 와 LARY 방식으로 정렬(Alignment) 학습시킨다면 데이터 효율을 극대화할 수 있습니다.
- **심미적 이동**: Latent Space가 잘 정렬되면 동작의 불연속성이 줄어들어, 보다 자연스럽고 부드러운(Smooth) 이동 궤적 생성이 가능해집니다.

---

# 11. 가치 점수 (Value)
- **가치: 4.5 / 5.0**
- **상세 이유**: 휴머노이드 이동 연구에서 가장 큰 난제 중 하나는 "자연어 명령 - 시각 정보 - 보행 동작" 사이의 괴리입니다. 이 논문은 이를 **Latent Space Alignment** 라는 관점에서 해결하려 하며, 이는 단순한 End-to-End 학습보다 훨씬 일반화 가능성이 높은 접근법입니다. 특히 이동 특화 VLA 모델 설계 시, Action Space를 어떻게 정의할지에 대한 강력한 힌트를 제공합니다.

---

## [참고 원문 및 링크 모아보기]
- **논문 원문**: LARY: A Latent Action Representation Yielding Benchmark for Generalizable Vision-to-Action Alignment (https://arxiv.org/abs/2404.11689)
- **핵심 섹션**: 
    - `3.1 Model Architecture`: 모델 구조 및 CLIP 활용 상세.
    - `3.4 Loss Functions`: Vision-Action 정렬을 위한 손실 함수 설계.
    - `4.2 Experimental Results`: 벤치마크 성능 및 일반화 지표.
- **GitHub**: (논문 내 명시된 경우 확인 필요, 통상적으로 arXiv 페이지 참조) `https://github.com/Meituan-Robotics/LARY` (예시 URL, 실제 존재 여부는 논문 업데이트 확인 필요)
